stats/view.go

```diff
@@ -43,14 +43,14 @@ type View struct {
 
 	// ss are the channels through which the collected views data for this view
 	// are sent to the consumers of this view.
-	ss map[chan *ViewData]subscription
+	// ss map[chan *ViewData]subscription
 
 	// boolean to indicate if the the view should be collecting data even if no
 	// client is subscribed to it. This is necessary for supporting a pull
 	// model.
 	isForcedCollection bool
 
-	// TODO(jbd): Guard isForcedCollection.
```

Comment:
It looks like you still need to protect the isForcedCollection, because this it is mutable correct?

Reply:
Correct, it is not mutated by the event loop. We need to guard it.